setup.py

```diff
@@ -133,8 +133,10 @@ def _parse_requirements_file(file_path):
     "opencv-python<=4.6.0.66",
 ]
 _openpifpaf_integration_deps = [
-    "openpifpaf==0.13.6",
+    "openpifpaf==0.13.11",
```

Comment:
Would be great to also update this in SparseML. I am very confident that a minimal amount of work is needed.